src/main/java/com/thealgorithms/conversions/AffineConverter.java

```diff
@@ -0,0 +1,23 @@
+package com.thealgorithms.conversions;
```

Comment:
Maybe it would be more clear and structured to have its own package for this (eg units) inside the conversions? And also number systems (eg numsystems)? Because these classes pretty special "class" of convertors in many other converters. And in the future, if someone wants to add something, it will be easier to find.

Reply:
@vil02 what do you think?